content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,529 @@
+
+1. Navigate to Datadog [Log Forwarding][5].
+1. Click **Add a new archive**.
+1. Enter a descriptive archive name.
+1. Add a query that filters out all logs going through log pipelines so that those logs do not go into this archive. For example, add the query `observability_pipelines_read_only_archive`, assuming that no logs going through the pipeline have that tag added.
+1. Select **AWS S3**.
+1. Select the AWS Account that your bucket is in.
+1. Enter the name of the S3 bucket.
+1. Optionally, enter a path.
+1. Check the confirmation statement.
+1. Optionally, add tags and define the maximum scan size for rehydration. See [Advanced settings][6] for more information.
+1. Click **Save**.
+
+See the [Log Archives documentation][7] for additional information.
+
+## Install the Observability Pipelines Worker
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+The Observability Pipelines Worker Docker image is published to Docker Hub [here][1].
+
+1. Download the [sample pipeline configuration file][2].
+
+2. Run the following command to start the Observability Pipelines Worker with Docker:
+
+    ```shell
+    docker run -i -e DD_API_KEY=<API_KEY> \
+      -e DD_OP_PIPELINE_ID=<PIPELINE_ID> \
+      -e DD_SITE=<SITE> \
+      -e AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID> \
+      -e AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY> \
+      -p 8282:8282 \
+      -v ./pipeline.yaml:/etc/observability-pipelines-worker/pipeline.yaml:ro \
+      datadog/observability-pipelines-worker run
+    ```
+    
+    Replace `<API_KEY>` with your Datadog API key, `<PIPELINES_ID>` with your Observability Pipelines configuration ID, and `<SITE>` with {{< region-param key="dd_site" code="true" >}}. Replace `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` with the AWS credentials you created earlier. `./pipeline.yaml` must be the relative or absolute path to the configuration you downloaded in Step 1.
+  
+[1]: https://hub.docker.com/r/datadog/observability-pipelines-worker
+[2]: /resources/yaml/observability_pipelines/archives/pipeline.yaml
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+1. Download the [Helm chart][1] for AWS EKS.
+
+2. In the Helm chart, replace the `datadog.apiKey` and `datadog.pipelineId` values to match your pipeline and use {{< region-param key="dd_site" code="true" >}} for the `site` value. Replace `${DD_ARCHIVES_SERVICE_ACCOUNT}` in the Helm config with the service account you created earlier. Then, install it in your cluster with the following commands:
+
+    ```shell
+    helm repo add datadog https://helm.datadoghq.com
+    ```
+    ```shell
+    helm repo update
+    ```
+    ```shell
+    helm upgrade --install \
+        opw datadog/observability-pipelines-worker \
+        -f aws_eks.yaml
+    ```
+
+[1]: /resources/yaml/observability_pipelines/archives/aws_eks.yaml
+{{% /tab %}}
+
+{{% tab "APT-based Linux" %}}
+1. Run the following commands to set up APT to download through HTTPS:
+
+    ```
```

Comment:
Non-blocker: are sudo command also shell commands? ```shell

Reply:
I'll add `shell` in the next iteration because I'll need to separate out all of the commands.